### PR TITLE
chore: update npm to current version

### PR DESCRIPTION
**Pull request by ck319**
**What type of PR is this?**

**What this PR does / why we need it**:
Update NPM version to current version
**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by huntergregory — APPROVED** [review]

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-npm:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.4.29"
+        "v1.4.32"
```
Comment (mattstam) [inline vhdbuilder/packer/components.json:90]: Probably want to keep `v1.4.29` in there for a while too, and remove only once AKS has stopped pulling it in all regions.

**Review by mattstam — CHANGES_REQUESTED** [review]

File: vhdbuilder/packer/components.json (hunk 1 above)
Comment (ck319) [reply vhdbuilder/packer/components.json:91]: ok, updated

**Review by mattstam — APPROVED** [review]
Let me know when status checks pass and I will merge 🚀

**Comment by cameronmeissner** [comment]
feel free to reopen without using a fork if needed

**Pull request closed without merging**